tools/SetupFlow/DevHome.SetupFlow/Views/ReviewView.xaml

```diff
@@ -29,6 +29,8 @@
                     </Style>
                 </ResourceDictionary>
             </ResourceDictionary.MergedDictionaries>
+            <converters:EmptyObjectToObjectConverter x:Key="NonEmptyObjectToObjectVisibilityConverter" NotEmptyValue="Visible" EmptyValue="Collapsed"/>

```

Comment:
NIT: remove the second "Object" in both converters.  `NonEmptyObjectToVisibilityConverter` or, to be more descriptive `EmptyObjectToHiddenConverter`